docs/reference/operator/api.adoc

```diff
@@ -148,6 +148,7 @@ Output defines a destination for log messages.
 |loki|object|  *(optional)* 
 |googleCloudLogging|object|  *(optional)* 
 |splunk|object|  *(optional)* 
+|http|object|  *(optional)* 
```

Comment:
is this a manual edit to this file or generated from the make target?

Reply:
perhaps it was generated from Makefile